src/web3/connectors.ts

```diff
@@ -77,7 +75,7 @@ const walletconnect = {
 //   },
 // };
 
-export const getConnectors = (chainId: number): Connectors => {
+export const getConnectors = (chainId: number): object => {
```

Comment:
Actually it's used here:

```
const migrateLocalStorage = (): void => {
  const version = LocalStorage.get('version');

  if (version === 0) {
    const connectorId = LocalStorage.get<'connectorId'>('connectorId');

    if (connectorId) {
      // The injected connector now requires a subType (MetaMask/Brave/etc),
      // so we can't assume the type; these users will have to connect again.
      if (connectorId !== 'injected') {
        const subType = CONNECTORS.find(c => c.id === connectorId)?.subType;
        LocalStorage.set('connector', {
          id: connectorId,
          subType,
        });
      }
      LocalStorage.removeItem<'connectorId'>('connectorId');
    }

    LocalStorage.setVersion(1);
  }
```

Do we have to replace the `CONNECTORS` with `WALLETS` here? I assumed since this is an old version of local storage, we still have to reference the `CONNECTORS`. Curios if this is true

Reply:
Oh ok, then we can leave it in :+1: